StreamConsumer/README.md

```diff
@@ -439,3 +585,265 @@ Rapid pro requires a specific format for phone numbers and it's better to change
     ]
 }
 ```
+
+#### TB / Suspect Registration
+
+```json
+{
+  "id": "tb_1",
+  "name": " TB 1 / Suspect Registration",
+  "error_handling": {
+    "error_topic": "log_tb_1",
+    "log_failure": true,
+    "log_success": true
+  },
+  "kafka_subscription": {
+    "topic_pattern": "TBForm*",
+    "topic_options": {
+      "masking_annotation": "@aether_masking",
+      "masking_levels": [
+        "public",
+        "private"
+      ],
+      "masking_emit_level": "public",
+      "filter_required": false
+    }
+  },
+  "const": {
+    "rapidpro_flow": "https://api.textit.in/api/v2/flow_starts.json",
+    "rapidpro_auth_header": "Token API_KEY_GOES_HERE",
+    "rapidpro_flow_id": "f01df7ef-f0cc-493e-91fa-f9bae1f2c703"
+  },
+  "stages": [
+    {
+      "name": "urn",
+      "type": "jscall",
+      "id": "urn",
+      "transition": {
+        "input_map": {
+          "base": "$.source.message.contact_number"
+        },
+        "output_map": {
+          "result": "$.result"
+        }
+      }
+    },
+    {
+      "name": "rapidpro",
+      "type": "restcall",
+      "id": "default",
+      "transition": {
+        "input_map": {
+          "url": "$.const.rapidpro_flow",
+          "method": "POST",
+          "headers": {
+            "Authorization": "$.const.rapidpro_auth_header"
+          },
+          "json_body": {
+            "flow": "$.const.rapidpro_flow_id",
+            "extra": {
+              "msg": "$.source.message.sms_body",
+              "sms_to_driver": "$.source.message.sms_to_driver"
+            },
+            "urns": "$.urn.result"
+          }
+        },
+        "output_map": {
+          "status_code": "$.status_code",
+          "all": "$.reason"
+        }
+      }
+    }
+  ]
+}
+```
+
+
+#### TB / Health care worker interpretation of TB test result
+
+```json
+{
+  "id": "tb_8",
+  "name": "TB 8 / Health care worker interpretation of TB test result",
+  "error_handling": {
+    "error_topic": "log_tb_8",
+    "log_failure": true,
+    "log_success": true
+  },
+  "kafka_subscription": {
+    "topic_pattern": "TBForm*",
```

Comment:
This should be specific to the form like `TBForm8*` so you can catch future versions like `TBForm8-11`, right now both it and other TB pipeline are catching all TBForms. They shouldn't be competing since they have different pipeline IDs, but it's extra data that's being processes by the wrong pipeline.